src/app/dashboard/home/home.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { User } from 'src/app/interfaces/user';
+import { UserService } from 'src/app/services/user.service';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.css'],
+})
+export class HomeComponent implements OnInit, OnDestroy {
+  users: User[] = [];
+  usersSubscription: Subscription = new Subscription();
+
+  constructor(private userService: UserService) {}
+  ngOnInit() {
+    this.usersSubscription = this.userService.usersObservable.subscribe(
+      (users) => (this.users = users)
+    );
+  }
+
+  ngOnDestroy() {
+    this.usersSubscription.unsubscribe();
+  }
```

Comment:
One way to clean up subscription is how you have done it right now. Another way is to use the rxjs operator `takeuntil` to clean up all the subscriptions in a clean manner. Can you use the `takeuntil` operator in this exercise?